test/ToastNotificationTests/ToastNotificationTests.vcxproj.filters

```diff
@@ -38,5 +38,6 @@
     <CopyFileToFolders Include="$(TestPkgOutputFile)" />
     <CopyFileToFolders Include="$(TestPkgOutputPath)\Dependencies\$(PlatformTarget)\Microsoft.VCLibs.$(PlatformTarget)$(VCLibsDebugConfigName).14.00.Desktop.appx" />
     <CopyFileToFolders Include="$(TestPkgOutputFile)" />
+    <CopyFileToFolders Include="$(TestPkgOutputFile)" />
```

Comment:
Accidental duplication? (And it appears it was already duplicated between lines 38 and 40?)

Reply:
whoops not sure how that happened